EasyPost/Models/API/Claim.cs

```diff
@@ -0,0 +1,182 @@
+
+        /// <summary>
+        ///     The ID of the <see cref="Models.API.Insurance"/> object associated with this claim.
+        /// </summary>
+        [JsonProperty("insurance_id")]
+        public string? InsuranceId { get; set; }
+
+        // ReSharper disable once InconsistentNaming
+#pragma warning disable SA1300
+        [JsonProperty("payment_method")]
+        private string? _paymentMethod { get; set; }
+#pragma warning restore SA1300
+
+        /// <summary>
+        ///     The <see cref="ClaimPaymentMethod"/> of the claim.
+        /// </summary>
+#pragma warning disable SA1101
+        [JsonIgnore]
+        public ClaimPaymentMethod? PaymentMethod => _paymentMethod == null ? null : ValueEnum.FromValue<ClaimPaymentMethod>(_paymentMethod);
+#pragma warning disable SA1101
+
+        /// <summary>
+        ///     The name of the recipient of the reimbursement.
+        /// </summary>
+        [JsonProperty("recipient_name")]
+        public string? RecipientName { get; set; }
+
+        /// <summary>
+        ///     The amount that has been requested for reimbursement.
+        /// </summary>
+        [JsonProperty("requested_amount")]
+        public double? RequestedAmount { get; set; }
+
+        /// <summary>
+        ///     The salvage value of the damaged goods.
+        /// </summary>
+        [JsonProperty("salvage_value")]
+        public double? SalvageValue { get; set; }
+
+        /// <summary>
+        ///     The ID of the <see cref="Models.API.Shipment"/> associated with this claim.
+        /// </summary>
+        [JsonProperty("shipment_id")]
+        public string? ShipmentId { get; set; }
+
+        /// <summary>
+        ///     The current status of the claim.
+        /// </summary>
+        [JsonProperty("status")]
+        public string? Status { get; set; }
+
+        /// <summary>
+        ///     The reason for the current status of the claim.
+        /// </summary>
+        [JsonProperty("status_detail")]
+        public string? StatusDetail { get; set; }
+
+        /// <summary>
+        ///     The timestamp of the last status update.
+        /// </summary>
+        [JsonProperty("status_timestamp")]
+        public string? StatusTimestamp { get; set; }
+
+        /// <summary>
+        ///     The tracking code of the <see cref="Models.API.Shipment"/> associated with this claim.
+        /// </summary>
+        [JsonProperty("tracking_code")]
+        public string? TrackingCode { get; set; }
+
+        [JsonProperty("type")]
+        // ReSharper disable once InconsistentNaming
+#pragma warning disable SA1300
+        private string? _type { get; set; }
+#pragma warning restore SA1300
+
+        /// <summary>
+        ///     The <see cref="ClaimType"/> of the claim.
+        /// </summary>
+        [JsonIgnore]
+#pragma warning disable SA1101
+        public ClaimType? Type => _type == null ? null : ValueEnum.FromValue<ClaimType>(_type);
+#pragma warning restore SA1101
```

Comment:
I don't think we need to have a model class for this, we can just define it as a string.

Reply:
Since we are using the enum in the request, we want to remain consistent and use it in the response as well.